### ISSUE TITLE
Both fontello.css and fontello.scss are converted

### ISSUE THREAD

**Issue by ka8725** [railslove/fontello_rails_converter#33]
Having this exception in rails app:

```
It's not clear which file to import for '@import "fontello"'. Candidates: fontello.css fontello.scss Please delete or rename all but one of these files.
```
I think only one `fontello.css` or `fontello.scss` is enough.

**Comment by jhilden** [comment]
For rails apps you definitely want to use the `fontello.scss` file (which you should be able to get by explicitly do `@import "fontello.scss"`), but I'm trying to make the code more flexible for other environments/frameworks and they might want to use the pur CSS version.

Do you have a suggestion on which way it should behave?

**Comment by ka8725** [comment]
May be detect current environment?

```ruby
if defined?(Rails)
  generate_scss
else
  generate_css
end
```

**Comment by dlackty** [comment]
In our case we need to manually delete on of them to make Sprockets assets precompiling work.

**Comment by jhilden** [comment]
I changed master, so that the source `.css` stylesheet in favor of the `.scss` version.  At some point I might add an option to disable conversion if one is not in a Rails context.  It would be nice if you could try it out and report if it works as expected.

**Comment by dlackty** [comment]
@jhilden Left a comment on the commit. Could you please have a look?

**Comment by jhilden** [comment]
I just released version `0.4.2` which should fix this issue.  Let me know if something is still not working.
